src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
I agree. Currently this version is essentially the released version of our gapic generator: https://oss.sonatype.org/#nexus-search;gav~com.google.api~toolkit~~~~kw,versionexpand

We can probably bump up this version to 0.1.0 with the next release.